versioned_docs/version-3.x/developer/order-expiration.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Order Expiration
+---
+
+:::info
+
+expireOrdersAfter was introduced in **Saleor 3.13**
+
+deleteExpiredOrdersAfter was introduced in **Saleor 3.14**
+
+:::
+
+:::caution
+
+This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
+feedback. However, it is still undergoing development and is subject to modifications.
+
+:::
+
+## Introduction
+
+Saleor allows finalizing checkout without payment. This can be achieved by setting `orderSettings.allowUnpaidOrders` for a given `Channel` to `true` and calling `checkoutComplete` without any payment/transaction object or by using the `orderCreateFromCheckout` mutation with first order then payment flow.
```

Comment:
```suggestion
Saleor allows finalizing checkout without payment. This can be achieved in two ways:
- by setting `orderSettings.allowUnpaidOrders` for a given `Channel` to `true` and calling `checkoutComplete` without any payment/transaction object
- by using the `orderCreateFromCheckout` mutation with the first order and then payment flow.
```